pebblo/app/service/classification.py

```diff
@@ -0,0 +1,133 @@
+"""
+Module for prompt governance
+"""
+
+import traceback
+from enum import Enum
+from typing import Optional
+
+from pydantic import BaseModel, ConfigDict, Field, ValidationError
+
+from pebblo.app.libs.responses import PebbloJsonResponse
+from pebblo.app.models.models import AiDataModel
+from pebblo.entity_classifier.entity_classifier import EntityClassifier
+from pebblo.log import get_logger
+from pebblo.topic_classifier.topic_classifier import TopicClassifier
+
+
+class ClassificationMode(Enum):
+    ENTITY = "entity"
+    TOPIC = "topic"
+    ALL = "all"
+
+
+class ReqClassifier(BaseModel):
+    data: str
+    mode: Optional[ClassificationMode] = Field(default=ClassificationMode.ALL)
+    anonymize: Optional[bool] = Field(default=False)
+
+    model_config = ConfigDict(extra="forbid")
+
+
+logger = get_logger(__name__)
+topic_classifier_obj = TopicClassifier()
+
+
+class Classification:
+    """
+    Class for loader doc related task
+    """
+
+    def __init__(self, input: dict):
+        self.input = input
+        self.entity_classifier_obj = EntityClassifier()
```

Comment:
Is `EntityClassifier()` initialization needed inside init or can we move it outside same as `TopicClassifier()`?

Reply:
Moved it outside for consistency